Description: let Qt fall back to C++11/GCC atomics for MIPS
 MIPS assembly code for testAndSetRelaxed is currently broken, and
 Thiago Macieira suggested that we should just let Qt fall back to
 C++11/GCC atomics.
 .
 In Qt 5.7, all the processor-specific atomics will be removed, so
 we will be able to drop this patch.
 .
 See https://codereview.qt-project.org/121144 for details.
Author: Dmitry Shachnev <mitya57@debian.org>
Bug: https://bugreports.qt.io/browse/QTBUG-49168
Forwarded: not-needed
Last-Update: 2015-11-02

--- a/src/corelib/thread/qbasicatomic.h
+++ b/src/corelib/thread/qbasicatomic.h
@@ -60,8 +60,6 @@
 # include "QtCore/qatomic_armv5.h"
 #elif defined(Q_PROCESSOR_IA64)
 #  include "QtCore/qatomic_ia64.h"
-#elif defined(Q_PROCESSOR_MIPS)
-#  include "QtCore/qatomic_mips.h"
 #elif defined(Q_PROCESSOR_X86)
 #  include <QtCore/qatomic_x86.h>
 
